PharoTour/PharoTour.pillar

```diff
@@ -171,6 +171,11 @@ download. To download a remote image, select it and click the ==Create
 image== button (located on the top right, next to the ==Refresh template list==
 button).
 
+You can use your own local images with PharoLauncher, in addition to working
+with remote images. To do so, simply ensure that your .image and its associated
+.changes file are placed in a folder (with a name of your choosing) in your
```

Comment:
yeap thats correct , @aryehof you need to add that the image and changes files must be of the exact same name as the folder for this to work, add this commit and i will merge your PR